metrics-operator/controllers/common/analysis/types/types.go

```diff
@@ -0,0 +1,72 @@
+package types
+
+import "github.com/keptn/lifecycle-toolkit/metrics-operator/api/v1alpha3"
+
+type TargetResult struct {
+	FailureResult OperatorResult
```

Comment:
to be less aggressive in the naming 😅 
```suggestion
	FailedResult OperatorResult
```